x-pack/packages/kbn-entities-schema/.gitignore

```diff
@@ -0,0 +1 @@
+yarn.lock
```

Comment:
do we need this ? I don't see any yarn.lock or .gitignore in x-pack/packages/**

Reply:
hm i added because i had a local yarn lock file that i didn't want to commit. i can remove, but the fact there aren't any other yarn.lock files makes me think maybe we should just leave this here?

Reply:
my knowledge around this is quite limited - @elastic/kibana-core should we commit package level yarn.lock ?

Reply:
we didn't need it since dependencies have to be set on the root package.json